configure.ac

```diff
@@ -1201,6 +1210,32 @@ AS_IF([test "x$build_libuuid" = xyes], [
   AC_DEFINE(HAVE_LIBUUID, 1, [Define to 1 if you have the -luuid.])
 ])
 
+dnl
+dnl liblastlog2
```

Comment:
This should default to autodetection instead of enabling by default.

Reply:
I have set the default value to enable_liblastlog2=check and checking if sqlite3 is available.